Hymns/Favorites/RelmHelper.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
```

Comment:
I think you spelled the name wrong? RelmHelper.swift -> RealmHelper.swift

Reply:
Yup lol